extensions/VisualEditor/wikia/VisualEditor.hooks.php

```diff
@@ -60,4 +60,23 @@ public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $ou
 		return true;
 	}
 
+	/**
+	 * If the user has specified that they want to create a page with VE, add Inspectlet script.
+	 * @param Title $title Title being used for request
+	 * @param Article|null $article
+	 * @param OutputPage $output
+	 * @param User $user
+	 * @param WebRequest $request
+	 * @param MediaWiki $mediaWiki
+	 * @return bool Always true
+	 */
+	public static function onBeforeInitialize(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$article, $user, $mediaWiki". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)